src/views/juttle-view.js

```diff
@@ -417,6 +421,20 @@ var JuttleView = Base.extend({
             findFlattenedOptions(optionValidationConfig, '');
 
             return validFlattenedOptions;
+        },
+
+        getFlattenedParamValidationErrors: function(errs) {
```

Comment:
Do you mean I should move this method to be an instance one instead of a static one and stash the errors in the view before throwing? I think of this method as more of just a util. It happens to be in `JuttleView` because its related.

As we talked about, there is also the option of just throwing the errors flattened, but that feels a bit odd since they are not flat when passed into the constructor. This also somewhat gets at the question of how coupled to juttle should the views be.